helm/resource_release_test.go

```diff
@@ -57,6 +57,33 @@ func TestAccResourceRelease_basic(t *testing.T) {
 	})
 }
 
+func TestAccResourceRelease_import(t *testing.T) {
+	name := fmt.Sprintf("test-import-%s", acctest.RandString(10))
+	namespace := fmt.Sprintf("%s-%s", testNamespace, acctest.RandString(10))
+	// Delete namespace automatically created by helm after checks
+	defer deleteNamespace(t, namespace)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t, namespace) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckHelmReleaseDestroy(namespace),
+		Steps: []resource.TestStep{{
```

Comment:
Following on from my comment [here](https://github.com/terraform-providers/terraform-provider-helm/pull/394/files#diff-70e42ed4d5009a76bddc6fb6c16b913bR787). I would propose adding some additional checks that cover the case where we check for default values that haven't been set in the HCL.

Reply:
This test is currently failing and I'm in a bit of a deadlock right now cause I can't really figure out how to make it work.

Let me know if you have any ideas

Reply:
Think I've figured it out :)